src/NBitcoin.Tests/pos_transaction_tests.cs

```diff
@@ -170,7 +170,7 @@ public void CanSignTransaction()
                 ScriptSig = scriptPubKey
             });
             tx.AddOutput(new TxOut("21", key.PubKey.Hash));
-            Transaction clone = tx.Clone(network: Network.StratisMain);
+            Transaction clone = tx.Clone(Network.StratisMain.Consensus.ConsensusFactory);
```

Comment:
For all these tests, I think you can probably declare `Network network = Network.StratisMain;` somewhere and then use that, instead of repeating `Network.StratisMain.Consensus.ConsensusFactory` everywhere.

Reply:
OK I'll fix these 👍